README.md

```diff
@@ -2,4 +2,4 @@
 
 **Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```

Comment:
This README file is not as descriptive as it should be. Please gently use the template of the README.md file given in the course.

Reply:
- [ ] Keep doing the great work , really appreciate your feedback 👍